connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -939,6 +978,14 @@ export async function githubCodeSyncActivity({
   githubCodeRepository.lastSeenAt = codeSyncStartedAt;
   await githubCodeRepository.save();
 
+  await upsertDataSourceFolder({
```

Comment:
shouldn't that be the repo_code that's upserted here? where is that one upserted?
and the repo full would be in the top top of the repo sync?

Reply:
ok saw the upsert of repo code below
would put repo code at top of code sync activity, and repo full not in code sync activity but at the beginning of the repo full sync

Reply:
happy to elaborate if unclear :)